aap_chatbot/src/useChatbot/useChatbot.ts

```diff
@@ -363,8 +362,7 @@ export const useChatbot = () => {
         chatRequest.media_type = "application/json";
         await fetchEventSource(
           import.meta.env.PROD
-            ? //             ? "/api/lightspeed/v1/ai/streaming_chat/"
-              "/api/lightspeed/v1/ai/streaming_agent/"
```

Comment:
This was also added for hack-a-thon and even it is not correct for the final version of hack-a-thon code. I revert the change to point to the streaming_chat endpoint.